src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,27 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.13.0
+
+27 September 2023
+This version is compatible with Kubernetes versions 1.24 to 1.26.
+
+### Breaking Changes
+
+A valid license is now required for okteto to work. If you were running a previous version of okteto that has no license, an expired license or no license at all make sure to [reach out to us](https://www.okteto.com/contact/) to request one, [open a support ticket](https://okteto.com/support?_gl=1*1ikydzg*_ga*MTMyMjc0MTYyOS4xNjUyMjA1MjQ0*_ga_KSKZWJHTJZ*MTY5NTM5Njk3MS4yMTguMS4xNjk1Mzk3MDk3LjU2LjAuMA..) or signup for our [30 day free trial](https://www.okteto.com/free-trial/).
```

Comment:
I think we should also mention that when the trial license expire, the instance is locked.

Reply:
Agree, that is also a breaking change and it should be included